operatorconfig/driverconfig/common/k8s-1.26-values.yaml

```diff
@@ -21,7 +21,4 @@ images:
 
   # "images.externalhealthmonitorcontroller" defines the container images used for the csi external health monitor controller
   # container.
-  externalhealthmonitorcontroller: k8s.gcr.io/sig-storage/csi-external-health-monitor-controller:v0.7.0
-  
-  # "images.sdcmonitor" defines the container images used to monitor sdc container
```

Comment:
@panigs7 sdcmonitor related sample is deleted. Please check.

Reply:
Added it.